prime-dotnet-webapi/Services/OrganizationClaimService.cs

```diff
@@ -0,0 +1,42 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.EntityFrameworkCore;
+
+namespace Prime.Services
+{
+    public class OrganizationClaimService : BaseService, IOrganizationClaimService
+    {
+        public OrganizationClaimService(
+            ApiDbContext context,
+            IHttpContextAccessor httpContext)
+            : base(context, httpContext) { }
+
+        public async Task<bool> DoesOrganizationHaveClaimAsync(int organizationId)
+        {
+            return await _context.OrganizationClaims
+                .AsNoTracking()
+                .AnyAsync(oc => oc.OrganizationId == organizationId);
+        }
+
+        public async Task<bool> DeleteClaimAsync(int organizationId, int partyId)
```

Comment:
The thing about delete(s) is that usually a successful delete and the thing not existing are the same. In the end, the thing doesn't exist. Does this need to report success/fail?

Reply:
Also, I feel it would be better /safer to delete by ClaimId, rather than OrdId + PartyId